src/main/java/racingcar/car/Car.java

```diff
@@ -32,4 +32,13 @@ public int getPosition() {
 	public String getName() {
 		return this.name;
 	}
+
+	@Override
+	public int compareTo(Car otherCar) {
+		return Integer.compare(this.position , otherCar.getPosition());
+	}
```

Comment:
getter를 쓰신 이유가 있을까요? 같은 객체이기 때문에 직접 접근하면 어떨까요 :) ?
```suggestion
	public int compareTo(Car otherCar) {
		return Integer.compare(this.position , otherCar.position);
	}
```